tools/Dockerfile

```diff
@@ -0,0 +1,15 @@
+
+FROM        ocaml/opam2:debian-9-ocaml-4.06
+MAINTAINER  Christian Lindig <christian.lindig@citrix.com>
+
+RUN sudo apt-get install -qq -yy \
+  debianutils hwdata libdlm-dev libffi-dev libgmp-dev libnl-3-200   \
+  libnl-route-3-200 libpam-dev libpci-dev libssl-dev libsystemd-dev \
+  libxen-dev linux-libc-dev m4 perl pkg-config python uuid-dev
```

Comment:
If you want to make the image smaller you could do a `&& sudo apt-get clean` in the same RUN line (doing it on a separate RUN line would not have any benefits as the previous RUN line would've already created a layer in the docker image)

Reply:
Wouldn't the depext below already take care of installing most of the above though? Although I can see the benefit in caching even if it means having duplicate info, so lets keep it for now.

Reply:
The apt-get could be removed. It is is from the time when `depext` did not work. I'll add an `apt-get clean`.